running-a-nats-service/environment.md

```diff
@@ -11,24 +11,20 @@ If you do use load balancers you just need to understand the potential issues wi
 
 If routes or gateway connections go through load balancers then you could very well have the same problems as above, which could results in JetStream lost quorum periods and create undue re-synchronization and protocol overhead traffic.
 
-## Virtualization and its effect on maximum performance
+## Virtualization, Containerization
 
-NATS is 'cloud native' and expected to be deployed in virtual environments. However, when it comes to maximum performance, such as when running benchmarks in order to estimate the size of future production deployments, it is good to keep a few things in mind.
+NATS is 'cloud native' and expected to be deployed in virtual environments and/or containers.
 
-Think of Core NATS servers are a software equivalent of network switches. Enable JetStream, and they also become a new kind of DataBase server as well. 
+However, when it comes to ensuring the highest possible level of performance that NATS can provide it is good to keep a few things in mind.
+
+Think of Core NATS servers are a software equivalent of network switches. Enable JetStream, and they also become a new kind of DB server as well. 
 
 What you need to keep in mind is that when selecting the instance types and storage options for your NATS server host instances that in public clouds: *you get what you pay for*.
```

Comment:
added `keep in mind repetition`.

Reply:
You mean you don't like the usage of 'keep in mind' twice? If so I changed the second one to a 'remember'